docs/zz_generated.seed.yaml

```diff
@@ -146,6 +146,10 @@ spec:
             rhel: ""
             sles: ""
             ubuntu: ""
+          # PeEndpoint to use for accessing Nutanix Prism Element. No protocol or port should be passed,
+          peEndpoint: ""
+          # Optional: PePort to use when connecting to the Nutanix Prism Element endpoint (defaults to 9440)
+          pePort: null
```

Comment:
Let's keep the generated value the default port, same as we did with `port`. Needs changes here: https://github.com/kubermatic/kubermatic/blob/4f4928f3501e5aea22b57d1ed292ade3ea6e5a6f/codegen/example-yaml/main.go#L165.